.github/PULL_REQUEST_TEMPLATE_ANSIBLE_ROLE.md

```diff
@@ -0,0 +1,36 @@
+## Definition of Done
+
+**Note:** If any of the following are not done at the time of PR review, the PR should only be accepted if it links to appropriate issues on the backlog for that repo that covers the incomplete criteria.
+
+- Repo name:
+  - [ ] Prefixed with `ansible-role-`
+  - [ ] Noun based (e.g. `nginx`) not verb based (e.g. `build-nginx`)
+  - [ ] Aligns with convention for SAP-related roles (e.g. `ansible-role-sap-hana-backup`)
+- Repo meta data/config:
+  - [ ] Tagged with `ansible-role`
+  - [ ] Description consistent for service catalogue (e.g. `Centiq’s Ansible role for managing ...`)
+  - [ ] Centiq Staff have write access
+  - [ ] Centiq Service Accounts have read access
+- [ ] Standard ansible role `.gitignore`
+- [README](README.md) contains:
+  - [ ] Level 1 heading title of the form: `Ansible Role: ...`
+  - [ ] Build status badge for the `master` branch just beneath the title
+  - [ ] High-level description of the role (no more than a paragraph)
+  - [ ] All main subsections:
+    - [ ] `Requirements`
+    - [ ] `Role Variables` with defaults
+    - [ ] `Role Testing` with detail of how to test
+    - [ ] `Example Playbook` with a working example (or a link to one)
+    - [ ] `License` set to `Centiq Commercial in Confidence Source Code License` that links to the correct license file
+    - [ ] `Author Information` set and linked to the `Centiq Engineering Team`
+- [Travis Build](.travis.yml):
+  - [ ] Exists and is passing in Travis
+  - [ ] Running either:
+    - `molecule test`
+    - `molecule lint` (with a backlog issue for moving to `molecule test`)
```

Comment:
Is there always a need for a backlog issue to move to `molecule test` ? For example with the VMware roles we aren't likely to mock a vCenter, would it ever more to a molecule test rather than just linting?

Reply:
Great point - I guess it could alternatively include a note that justifies not having `molecule test`